### PR TITLE
Added docs for the ReplayTdsLogs tool

### PR DESCRIPTION
**Pull request by lesserwhirls**
Fixes Unidata/thredds#156

### CODE REVIEW

File: docs/website/tds/reference/ReplayLogs.adoc [hunk 1]
```diff
@@ -0,0 +1,72 @@
+:source-highlighter: coderay
+:linkcss:
+:stylesheet: ../tds.css
+
+= Testing a new version of the TDS
+
+Upgrading to a new version of the TDS can be a daunting task, especially when trying to figure out if your data collections are still accessible as your users expect.
+Perhaps one of the best "tests" a TDS administrator can do, although certainly no 100% complete, is to replay old user requests against a new, parallel test server to assess any issues. Note that the overhead of running a parallel test instance of Tomcat should be quite small, especially if the server is restricted to internal users only.
```
Comment (dopplershift) [inline docs/website/tds/reference/ReplayLogs.adoc]: `no` -> `not`

File: docs/website/tds/reference/ReplayLogs.adoc [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+----
+java -classpath toolsUI.jar thredds.logs.ReplayTdsLogs http://testmytds.me.org old_logs/
+----
+
+Follow the progress and allow the tool to complete.
+
+=== Examine the logs on the test server
+
+The final step is to examine the logs on the test server.
+This includes examining the newly created access logs for any issues (i.e. response codes of 404s, 500s, etc. that were note in the initial access logs used for testing), as well as looking for issues within the TDS logs.
```
Comment (dopplershift) [inline docs/website/tds/reference/ReplayLogs.adoc]: `note` -> `not`

File: docs/website/tds/reference/ReplayLogs.adoc (hunk 1 above)
Comment (lesserwhirls) [inline docs/website/tds/reference/ReplayLogs.adoc]: got it - thanks @dopplershift!

**Comment by dopplershift** [comment]
Make sure you get the last typo (which really did affect my ability to understand it). Otherwise, looks good!

**Comment by lesserwhirls** [comment]
Just fixed the last one. Thanks for the review!

**Pull request merged**